desktop/apps/artwork/components/clock/view.coffee

```diff
@@ -5,10 +5,13 @@ template = -> require('./index.jade') arguments...
 
 module.exports = class ClockView extends Backbone.View
   initialize: ({ @label, @timestamp }) ->
-    endAt = moment.utc @timestamp
+    endAt = moment.utc(@timestamp)
 
     if endAt.isAfter()
-      setTimeout (-> location.reload()), endAt.diff moment.utc()
+      reloadMs = Math.min(endAt.diff(moment.utc()), 0x7FFFFFFF)
```

Comment:
Just curious, what does `0x7FFFFFFF` represent here?

Reply:
The maximum value `setTimeout` will accept before firing immediately. I'll add a comment...